guides/doc-Release_Notes/topics/katello.adoc

```diff
@@ -4,14 +4,34 @@
 [id="katello-headline-features"]
 == Headline Features
 
-There are no highlights with Katello {KatelloVersion}.
+=== New All Hosts index page improvements ===
+* On the experimental new All Hosts index page, you can now click 'Manage Columns' and add Katello-provided columns, including
+** RHEL Lifecycle status
+** Installable updates
+** Last seen
+** Lifecycle environment
+** Content view
+** Content source
+** Registered at
+* Also, you can now change the content view and lifecycle environment of multiple hosts. Select some hosts, click the vertical ellipsis, then click 'Change content view environments.'
+
+=== New manifest expiration warnings ===
+* Users will now be notified before their subscription manifest expires. The number of days of notice is configured by the expire_soon_days setting.
+* If a manifest is expired or expiring soon, you'll see new alert banners on the Manage Manifest screen.
+* The expiration date of the manifest is now displayed on the Manage Manifest screen.
+* Refreshing your manifest will now extend the expiration date of the manifest to 1 year from the current date.
+
+=== Repair content view versions ===
+* A new hammer subcommand to verify checksums and redownload corrupted packages belonging to a content view version
+
+=== Other enhancements ===
+* Subscription expiration information has now moved from the Subscription - Entitlement report to the Subscription - General report. This report now accepts the expiring_in_days input, so you can see when your subscriptions will expire.
+* The activation keys selection on the Hostgroup Edit page has been refactored from jquery to React, providing a more modern look and user experience
+
 
 [id="katello-upgrade-warnings"]
 == Upgrade Warnings
 
```

Comment:
I think we should add:

- The SHA1 repository checksum type option has been removed in favor of newer, and more secure algorithms. SHA1 content can still be synced, but synced content can no longer be published using the SHA1 checksum type.

https://projects.theforeman.org/issues/37522